deployment/helm/skaha/skaha-config/launch-notebook.yaml

```diff
@@ -25,12 +25,29 @@ spec:
       imagePullSecrets:
         - name: "${software.imagesecret}"
       securityContext:
-        # runAsUser: ${skaha.posixid} 
-        # runAsGroup: ${skaha.posixid}
+        runAsUser: ${skaha.posixid} 
+        runAsGroup: ${skaha.posixid}
         fsGroup: ${skaha.posixid}
         supplementalGroups: [${skaha.supgroups}]
       priorityClassName: uber-user-preempt-medium
       hostname: "${software.hostname}"
+      initContainers:
+      - name: init-notebook
+        image: ${software.imageid}
+        command: ["/skaha-system/init-jupyterlab.sh"]
+        args:
+        - ${user.posix.entry}
+        - ${user.group.entry}
+        volumeMounts:
+        - mountPath: "/etc-passwd"
+          name: etc-passwd
+        - mountPath: "/etc-group"
+          name: etc-group
+        - mountPath: "/skaha-system"
+          name: start-jupyterlab
+        securityContext:
```

Comment:
The container securityContext is redundant and could be removed since the securityContext of the pod applies by default, see `kubectl explain pod.spec.containers.securityContext`